### PR TITLE
Update periodic webui e2e workflow

### PR DESCRIPTION
**Pull request by rvykydal**
Run it nightly on master.
Testing (PASSING!) run on my fork: https://github.com/rvykydal/anaconda/actions/runs/5877341654

We may need to de-duplicate the content with e2e tests on PR (https://github.com/rhinstaller/anaconda/blob/master/.github/workflows/webui-tests.yml) instead of the copy&paste but this fruit is hanging so low that I think it is worth to do it even by copying now. Plus I don't have nice memories regarding reusing of workflow parts...

### CODE REVIEW

File: .github/workflows/webui-periodic.yml.j2 [hunk 1]
```diff
@@ -56,54 +56,105 @@ jobs:
...
 
-      - name: Run webui integration tests
+      - name: Run webui end2end tests
         working-directory: ./permian
         run: |
           PYTHONPATH=${PYTHONPATH:-}:${{ github.workspace }}/tplib \
           ./pipeline --debug-log permian.log \
             --settings settings.ini \
-            --override workflows.dry_run=False \
+            -o github.pull-request=${{ github.event.issue.number }} \
```
Comment (velezd) [inline .github/workflows/webui-periodic.yml.j2:116]: github-pr report sender is not used (there is no pull request) so these options are not doing anything

File: ui/webui/test/end2end/periodic.plan.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-name: WebUI Integration daily preview
-description: Runs all WebUI integration tests on preview image periodically
+name: WebUI end2end periodic tests
```
Comment (velezd) [inline ui/webui/test/end2end/periodic.plan.yaml:1]: You have to add xunit report sender

File: .github/workflows/webui-periodic.yml.j2 [hunk 3]
```diff
@@ -56,54 +56,105 @@ jobs:
...
+            permian/xunit*.xml
             permian/*.dump
+
+      # Permian exit code doesn't reflect the test results
+      - name: Check tests results
+        if: always()
+        working-directory: ./permian
+        shell: bash {0}
+        run: |
+          fails=$( grep -E '<failure .* type="failure"/>|<error .* type="error"/>' xunit*.xml )
```
Comment (velezd) [inline .github/workflows/webui-periodic.yml.j2:146]: Looks like this check should be improved. In your test it failed to catch that there are no xunit files https://github.com/rvykydal/anaconda/actions/runs/5877341654/job/15937287215
This applies to webui-tests on PR too.

**Review by velezd — CHANGES_REQUESTED** [review]

File: .github/workflows/webui-periodic.yml.j2 (hunk 1 above)
Comment (rvykydal) [reply .github/workflows/webui-periodic.yml.j2]: Updated

File: ui/webui/test/end2end/periodic.plan.yaml (hunk 2 above)
Comment (rvykydal) [reply ui/webui/test/end2end/periodic.plan.yaml:1]: Added

File: .github/workflows/webui-periodic.yml.j2 (hunk 3 above)
Comment (rvykydal) [reply .github/workflows/webui-periodic.yml.j2:149]: Updated (also a commit for the tests on PR).

**Comment by rvykydal** [comment]
Test run of the updated version: https://github.com/rvykydal/anaconda/actions/runs/5879419693

**Review by VladimirSlavik — APPROVED** [review]
Thank you, it's great to have it work!

**Comment by rvykydal** [comment]
> Thank you, it's great to have it work!

Thank you for creating the workflow I was copy&pasting from:)

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**